website/redirects.js

```diff
@@ -1381,4 +1381,16 @@ module.exports = [
     destination: '/api-docs/:path*',
     permanent: true,
   },
+
+  // Redirects for exact versioned-docs to respective generic versions
+  {
+    source: '/:base(docs|api-docs)/v1.:minor([0-2]{1,}).(\\d)',
```

Comment:
```suggestion
    source: '/:base(docs|api-docs)/v1.:minor([0-2]{1,}).(\\d+)',
```

I think we need match multiple digits after the minor?

Reply:
@lgfa29 I think you're right... but the redirect still appeared to work without `+`, according to the tests I ran in the PR description...

Reply:
I'll add commit your suggestion and verify again!

Reply:
Ah interesting. I guess there will be a prefix match regardless, since there's no `$` to represent the end of the string so the original expression looks good 👍

Reply:
Also looks good:

```
curl -I -X GET https://nomad-9q62puzgk-hashicorp.vercel.app//docs/1.2.9000
HTTP/2 308
```

I'll keep your suggestion!